### PR TITLE
Password update function on user edit page

### PR DESCRIPTION
**Pull request by kamilczaja**
```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md and linked the changes to their issues. See [changelog_update.md](https://github.com/sovity/authority-portal/blob/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: authority-portal-frontend/src/app/core/services/config/app-config.ts [hunk 1]
```diff
@@ -70,6 +71,7 @@ export interface AppConfigEnv {
   AUTHORITY_PORTAL_FRONTEND_BACKEND_URL: string;
   AUTHORITY_PORTAL_FRONTEND_LOGIN_URL: string;
   AUTHORITY_PORTAL_FRONTEND_LOGOUT_URL: string;
+  AUTHORITY_PORTAL_FRONTEND_UPDATE_PASSWORD_URL: string;
```
Comment (PaddseL) [inline authority-portal-frontend/src/app/core/services/config/app-config.ts:74]: Add documentation for this to the [user documentation](https://github.com/sovity/authority-portal/tree/main/docs/product/user-documentation), the [deployment README.md](https://github.com/sovity/authority-portal/blob/main/docs/deployment-guide/goals/production/README.md) and the [Deployment Migrations Notes-section of the CHANGELOG.md](https://github.com/sovity/authority-portal/blob/main/CHANGELOG.md#deployment-migration-notes)?

**Comment by PaddseL** [comment]
Is it correct that you don't have to confirm your current password when changing the password?

**Review by PaddseL — CHANGES_REQUESTED** [review]
Functionality looks good to me, suggested some minor changes and added a general question.

**Comment by kamilczaja** [comment]
> Is it correct that you don't have to confirm your current password when changing the password?

Keycloak decides for itself if it wants to re-authenticate. It did prompt me for the old password + OTP on some locations so I assume it determines that depending on the auth token age or something

**Review by PaddseL — APPROVED** [review]
LGTM

**Pull request merged**